README.md

```diff
@@ -70,17 +73,18 @@ If you want to contribute but are unsure how to start, we have [a "good first is
 
 #### Technologies
 
-With the ground rules out of the way, let's talk about the coarse architecture of this mono repo:
+Let's discuss the key technologies that power this monorepo, providing a foundation for our healthcare application development:
+
+- **Full-stack TypeScript**: Our codebase is primarily built using TypeScript, enabling type-safe development across both the server and frontend components. Node.js is used for server-side code, while React powers the frontend applications.
 
-- **Full-stack TypeScript**: We use Node.js to power our servers, and React to power our frontend apps. Almost all of the code you'll touch in this codebase will be TypeScript.
+Here is a list of the major technologies in use:
 
-Here is a list of all the big technologies we use:
+- **PostgreSQL**: Our choice for data storage, ensuring a robust and reliable foundation for healthcare data.
+- **Redis**: Used for background jobs and caching, enhancing the performance and responsiveness of our applications.
+- **Express**: Serving as our API server, Express simplifies routing and handling requests, facilitating efficient data exchange.
```

Comment:
"Serving as [a] server" sounds a bit tautological to me — consider something like this instead:
```suggestion
- **Express**: Serving as our API framework, Express simplifies routing and handling API requests
```